source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -213,6 +233,23 @@ content: |
 
            data_key_id = create_data_encryption_key()
 
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: csharp
+
+           var keyVaultClient = new MongoClient(_connectionString);
+           var clientEncryptionOptions = new ClientEncryptionOptions(
+               keyVaultClient: keyVaultClient,
+               keyVaultNamespace: _keyVaultNamespace,
```

Comment:
Just curious to learn what underscore is used for in _keyVaultNamespace and _connectionString, is it special C#-related syntax that modifies the original variable name somehow?

Reply:
I'm not sure. During code review with dima these were suggested. I'm guessing it's just more idiomatic.